### DISCUSSION TITLE
The wallpaper on the lock screen cannot be updated

### DISCUSSION THREAD

**Discussion by oliverricardo** [t1m0thyj/WinDynamicDesktop#573] [Q&A]
In the previous version, by right-clicking, you can choose to update the lock screen wallpaper, but now this version, there is no such option after right-clicking, how to make the lock screen wallpaper the same as the desktop wallpaper? My version is 5.4.2 (UWP)

**Comment by t1m0thyj** [comment]
This feature still exists but has been moved. See https://github.com/t1m0thyj/WinDynamicDesktop/discussions/571#discussioncomment-8618257
